ng/src/web/pages/alerts/component.js

```diff
@@ -366,9 +366,25 @@ class AlertComponent extends React.Component {
       if (is_defined(onTestSuccess)) {
         onTestSuccess(_('Testing the alert {{name}} was successful.', alert));
       }
-    }, () => {
+    }, response => {
```

Comment:
please use rejection as variable name here. that's clearer which object you will get.